src/App.js

```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
```

Comment:
clean this up, make sure you use the correct format

make this into a `jsx` file

Reply:
Cleaned up and suffix changed